src/app/components/change-request/change-requests.jsx

```diff
@@ -12,6 +12,12 @@ import {
 } from 'components/filter';
 import { AnalyticsContext, useAnalyticsContext } from 'shared/contexts';
 
+const useStyles = makeStyles(() => ({
```

Comment:
```suggestion
const useStyles = makeStyles({
```

We do not use the function version of this tool unless necessary